file_paths.md

```diff
@@ -305,6 +306,27 @@ The package will install a service named `puppetserver`, create a
         puppetserver/                     # writeable by puppetserver
             puppetserver.pid
 
+# puppetmaster
+
+The package will install a service named `puppetmaster`, create a
+`puppet` user and group, and run the service as the `puppet` user.
+
+    /opt/puppetlabs/server                # serverside apps live underneath
+        data
+            puppetmaster *                # :vardir (and $HOME for services that use it)
```

Comment:
@joshcooper so there is an oddity here. the package will create those paths, but not own them. do we care about that distinction? (if it owned /opt/puppetlabs/server, then no other server packages could also own them. it would get weird fast.).